### PR TITLE
Resolved Bug 625 : Design System Issues Of Elements and Charts

### LINKED ISSUE #625: Design System 7.6.20 > Issues of Elements, chart
Design System 7.6.20 > Issues of Elements, chart

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/TRIVEN~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/TRIVEN~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">

Sr.No | Elements/Components | Issue
-- | -- | --
  | Elements |  
1 | Button Group | Open Checkbox button   group in small mobile view its not show properly.
2 | Collapse | Open Collapse in   small and large mobile view its not show properly.
3 | Modal | In a dark theme   vertically centered modal close button text not show properly.
4 | Notification | in a small mobile   view "SET AS READ"  button   text not show properly.
5 | Pagination | in a dark theme drop   down number not show properly.
  |   | in a advanced small   mobile view pagination dropdown alignment    not show properly.
6 | Popover | in a dark theme   button "POPOVER" color not show properly.
7 | Select List | in a dark theme   dropdown text color not show properly.
  |   | click on multiple   option => in a dropdown select list option not selected.
8 | Textarea | in a dark theme   placeholder text color not show properly.
9 | Toast | Toast without header   close button alignment not show properly.
  |   |  
  | Charts |  
10 | Doughnut Chart | in this chart Blue   color not show proper color.
  |   |  
11 | Application  Shells | In a small mobile   view => click on user icon =>personal data text hide behind the button.

</body>

</html>

### PR DESCRIPTION
**Pull request by prakashmhaske**
## Description

Resolved Issue on Design System Issues Of Elements , Chart

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/PRAKAS~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/PRAKAS~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
</head>

<body link="#467886" vlink="#96607D">

Sr no | Elements/Component | Issue
-- | -- | --
1 | Button   Group | Open   Checkbox button group in small mobile view its not show properly.
2 | Collapse | Open   Collapse in small and large mobile view its not show properly.
3 | Modal | In   a dark theme vertically centered modal close button text not show properly.
4 | Notification | in   a small mobile view "SET AS READ"  button text not show   properly.
5 | Pagination | in   a dark theme drop down number not show properly.
  |   | in a advanced small mobile view pagination dropdown   alignment  not show properly.
6 | Popover | in   a dark theme button "POPOVER" color not show properly.
7 | Select List | in   a dark theme dropdown text color not show properly.
  |   | click   on multiple option => in a dropdown select list option not selected.
8 | Textarea | in a dark theme placeholder text color not show properly.
9 | Toast | Toast without header close button alignment not show properly.
10 | Doughnut Chart | in this chart Blue color not show proper color.
11 | Application  Shells | In a small mobile view => click on user icon =>personal   data text hide behind the button.

</body>

</html>

 
## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Ketan-Khalate** [comment]
@prakashmhaske you need to check storybook running locally as some changes for merged previously also so you have to confirm that taking latest into your branch and verify your changes and if needed if classes are already there just remove your existing changes.

cc :- @kaushikgokhale @trivenikulkarni @rameshwarkamandar

File: raaghu-elements/src/rds-select-list/rds-select-list.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.theme-dark .css-1dimb5e-singleValue {
+    color: white;
+}
```
Comment (Ketan-Khalate) [inline raaghu-elements/src/rds-select-list/rds-select-list.css]: @prakashmhaske  give proper class name and then apply css to it

Comment (prakashmhaske) [reply raaghu-elements/src/rds-select-list/rds-select-list.css]: Hello @Ketan-Khalate I have resolved comments Please check and approved
cc :- @kaushikgokhale , @trivenikulkarni ,

Comment (Ketan-Khalate) [reply raaghu-elements/src/rds-select-list/rds-select-list.css]: @prakashmhaske  give proper class in place of .raaghu-select__single-value 

cc - @trivenikulkarni @kaushikgokhale

Comment (prakashmhaske) [reply raaghu-elements/src/rds-select-list/rds-select-list.css]: I have resolved Please check @Ketan-Khalate

**Review by Ketan-Khalate — APPROVED** [review]

**Comment by Ketan-Khalate** [comment]
Changes Reviewed and PR Approved

**Pull request merged**